services/datamanager/builtin/capture_test.go

```diff
@@ -134,28 +160,22 @@ func TestDataCaptureEnabled(t *testing.T) {
 			// Update to new config and let it run for a bit.
 			err = dmsvc.Update(context.Background(), updatedConfig)
 			test.That(t, err, test.ShouldBeNil)
-			time.Sleep(captureTime)
-
-			// Check if initial config captured (or not) as we'd expect.
-			initialCaptureFiles := getAllFiles(initCaptureDir)
-			if !tc.initialServiceDisableStatus && !tc.initialCollectorDisableStatus {
-				// TODO: check contents
-				test.That(t, len(initialCaptureFiles), test.ShouldBeGreaterThan, 0)
-				test.That(t, initialCaptureFiles[0].Size(), test.ShouldBeGreaterThan, emptyFileBytesSize)
-			} else {
-				test.That(t, len(initialCaptureFiles), test.ShouldEqual, 0)
-			}
+			oldCaptureDirFiles := getAllFiles(initCaptureDir)
 
-			// Check if updated config captured (or not) as we'd expect.
-			updatedCaptureFiles := getAllFiles(updatedCaptureDir)
+			time.Sleep(captureTime)
 			if !tc.newServiceDisableStatus && !tc.newCollectorDisableStatus {
-				//TODO: check contents
+				updatedCaptureFiles := getCapturedFiles(updatedCaptureDir)
 				test.That(t, len(updatedCaptureFiles), test.ShouldBeGreaterThan, 0)
 				test.That(t, updatedCaptureFiles[0].Size(), test.ShouldBeGreaterThan, emptyFileBytesSize)
 			} else {
+				updatedCaptureFiles := getAllFiles(updatedCaptureDir)
 				test.That(t, len(updatedCaptureFiles), test.ShouldEqual, 0)
+				oldCaptureDirFilesAfterWait := getAllFiles(initCaptureDir)
+				test.That(t, len(oldCaptureDirFilesAfterWait), test.ShouldEqual, len(oldCaptureDirFiles))
+				for i := range oldCaptureDirFiles {
+					test.That(t, oldCaptureDirFiles[i].Size(), test.ShouldEqual, oldCaptureDirFilesAfterWait[i].Size())
+				}
 			}
-			test.That(t, dmsvc.Close(context.Background()), test.ShouldBeNil)
```

Comment:
~~[nit]: I feel like we should be still checking this~~
I see the defer above now!